src/Blosc.jl

```diff
@@ -4,7 +4,7 @@ module Blosc
 export compress, compress!, decompress, decompress!
 
 using Compat
-import Compat.String
+using Compat.String
```

Comment:
I don't see that `String` is being extended, so it's cleaner not to import it since we are not extending the method